cmd/cbnt-prov/cmd.go

```diff
@@ -430,14 +431,22 @@ func (g *generateBPMCmd) Run(ctx *context) error {
 			seg.Flags = g.IbbSegFlag
 			se.IBBSegments = append(se.IBBSegments, seg)
 		}
-		if g.Coreboot {
+		if g.Coreboot && !g.Uefi {
```

Comment:
With xor in the cmdline flags, can we just do 'if g.Coreboot {' ?

Reply:
true